src/Tools/GenerateAnalyzerNuspec/Program.cs

```diff
@@ -85,6 +85,7 @@
 result.AppendLine(@"  </metadata>");
 
 result.AppendLine(@"  <files>");
+result.AppendLine(@"    $CommonFileElements$");
```

Comment:
Added this based on the description in https://github.com/dotnet/arcade/pull/4078.